### PR TITLE
Multiple Updates

### LINKED ISSUE #266: When debug session ends, the binary display of hex data goes away
When debug session ends, the binary display of hex data goes away. Why? I might be very interested in looking at that still. It can be marked inactive like the infoset file is, but shouldn't just be closed. You may not want to fix this soon because I expect the hex display will be replaced by the data editor in a view mode at some point, but anyway, it shouldn't auto-close just because the parse ended.

### LINKED ISSUE #267: Config Wizard text didn't display in a useable way in light UI mode
Config Wizard - did not work for me - text didn't display in a way I could use this. With dark theme I could use it.

### PR DESCRIPTION
**Pull request by Shanedell**
Multiple Updates:

- Make launch wizard display properly when using light themes.
- No longer closer the hexview once debug has ended.

Closes #266
Closes #267

### CODE REVIEW

File: src/styles/styles.css [hunk 1]
```diff
@@ -86,7 +86,7 @@
 }
 
 .setting-div {
-  color: white;
+  /* color: white; */
```
Comment (scholarsmate) [inline src/styles/styles.css:89]: Don't bother commenting this out, just delete this line.

**Review by scholarsmate — CHANGES_REQUESTED** [review]

**Comment by scholarsmate** [comment]
@stevedlawrence and @mbeckerle, will you guys review this PR?  This addresses a couple of (minor) issues that Mike brought to our attention in his review of version 1.1.0 when it went up for a release vote.

**Review by stevedlawrence — APPROVED** [review]
+1

Note, In general I would really prefer a single PR per issue, even when they are small one liners. When you include multiple changes in a single PR the commit subject is something generic like "multiple updates" which is very unhelpful when looking back at history. I would much rather see two PRs and commits with appropriate commit messages.

**Comment by Shanedell** [comment]
> +1
> 
> Note, In general I would really prefer a single PR per issue, even when they are small one liners. When you include multiple changes in a single PR the commit subject is something generic like "multiple updates" which is very unhelpful when looking back at history. I would much rather see two PRs and commits with appropriate commit messages.

Yeah this makes sense I will make sure to do them as single PRs from now one.

**Comment by scholarsmate** [comment]
@mbeckerle, will you review this PR, we need your vote, and these address issues that you've found during the last release vote.

**Review by tuxji — APPROVED** [review]
+1

I suggest removing line 89 from style.css in a local commit and rebasing, squashing, and force-pushing the PR back to 1 commit with an improved commit message which says something like "Improve launch wizard display, don't close hexview" instead of "Multiple Updates".  Then git history will look better after you merge the PR.

**Comment by Shanedell** [comment]
@tuxji Does the commit look better?

**Review by mbeckerle — APPROVED** [review]
+1

**Review by scholarsmate — APPROVED** [review]
+1

Looks good, thanks!

**Pull request merged**